src/qcodes/instrument_drivers/stahl/stahl.py

```diff
@@ -62,11 +63,13 @@ def __init__(self, parent: VisaInstrument, name: str, channel_number: int):
         self._channel_string = f"{channel_number:02d}"
         self._channel_number = channel_number
 
+        FLOATING_POINT_RE = r"[+\-]?(?:[.,]\d+|\d+(?:[.,]\d*)?)(?:[eE][-+]?\d+)?"
```

Comment:
```suggestion
        _FLOATING_POINT_RE = r"[+\-]?(?:[.,]\d+|\d+(?:[.,]\d*)?)(?:[eE][-+]?\d+)?"
```

Lets make this one private by starting the name with an _. That would need to be fixed where its used too

Reply:
will do but it's a local variable so what do you mean make it private? do you want me to extract it?